src/lib/canvas/grid.js

```diff
@@ -29,88 +31,46 @@ const Controls = styled.div`
   font-size: 10px;
   z-index: 9999;
```

Comment:
don't like the approach with extream values for z-index just to put it on top

Reply:
actually, I want to don't spend time on this.
because I will completely remove this controls with tabs in the next part of this feature.

we decided to split all logic on 2 PR.  for QA testing. 
because they're a lot of cases for QA and me)